src/ck-core/init.C

```diff
@@ -1495,6 +1498,7 @@ void _initCharm(int unused_argc, char **argv)
 		*/
 		_registerCkFutures();
 		_registerCkArray();
+                _registerCkSyncBarrier();
```

Comment:
Not sure if you want to match the surrounding indentation here

Reply:
Good call, fixed.